static/css/style.css

```diff
@@ -39,8 +39,31 @@ body {
 -------------------------------------------------- */
 /* Not required for template or sticky footer method. */
 
+.login {
+	padding: 10px 0px 5px 45px; 
+}
+.h1 {
+	border: solid;
+	width: 350px;
+	text-align: center;
+	position: relative; 
+	margin:-100px 0 0 -150px;
+	top: 100px;
+	left: 50%;
+	font-family: 'Carter One', cursive;
+}
+.h2 {
+	color: #707070; 
+	text-align: center;
+	position: relative; 
+	top: 120px; 
+}
+.jumbotron {
+	text-align:center; 
+	height: 300px;
+}
 .container {
-	padding-top: 60px;
+	padding-top: 0px;
```

Comment:
This is nothing bad, but in CSS world usually one does not put any unit if something is defined with 0. I just thought you might want to know this, to become a badass in CSS ;)

`padding-top: 0;`